exodus/web/templates/base/base.html

```diff
@@ -69,7 +69,7 @@
         <form class="form-inline my-2 my-lg-0" action="{% url 'set_language' %}" method="post">
           <img class="mr-3 mr-lg-2 mr-xl-3" alt="language selector" src="{% static 'img/langue.svg' %}">
           {% csrf_token %}
-          <input name="next" type="hidden" value="{{ redirect_to }}">
+          <input name="next" type="hidden" value="{{ request.get_full_path|slice:'3:' }}">
```

Comment:
I would have added spaces around the pipe for better readability (and consistency)
`{{ request.get_full_path | slice:'3:' }}`